robottelo/constants/__init__.py

```diff
@@ -317,7 +319,7 @@ class Colored(Box):
 REPOS = {
     'rhel7': {
         'id': 'rhel-7-server-rpms',
-        'name': 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7Server',
+        'name': 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7.9',
```

Comment:
For this and the rhel 8 KS repos below, do we want to lock a certain x.y version and have to update it or just use latest? I see the rhel8_bos at the end of the `REPOS` dict has just the whole number `8` for the version. I'm assuming this would grab latest?

Reply:
@Griffin-Sullivan Kickstart repos are only available by specifying y-stream versions; there's no option to just use the latest version. However, with the non-KS repos, you're right that using `8` will grab the latest. This is also true for `7Server`, which should be functionally identical to `7.9`. Because of this, I was curious about this change. @shubhamsg199 This is a non-blocking question, but could you please explain your reasoning for switching from `7Server` to `7.9` here?

Reply:
Switched back to 7Server. I did this change during my local testing, forgot to change it back :)